internal/telemetry/event.go

```diff
@@ -34,16 +34,23 @@ const (
 
 // set of event data keys
 const (
-	eventDataKeyCommand     = "cmd"
-	eventDataKeyExecutionID = "xid"
-	eventDataKeyVersion     = "v"
-
-	EventDataKeyError = "err"
+	eventDataKeyCommand      = "cmd"
+	eventDataKeyExecutionID  = "xid"
+	eventDataKeyVersion      = "v"
+	eventDataKeyError        = "err"
+	eventDataKeyErrorMessage = "err_msg"
 
 	// EventDataKeyTemplate used to tracked if templates were used to create an app
 	EventDataKeyTemplate = "templateId"
 )
 
+func EventDataError(err error) []EventData {
+	return []EventData{
```

Comment:
should we check `if err != nil`? This will panic otherwise

Reply:
the usages of this function are already wrapped in `!= nil`, but the better-safe-than-sorry approach seems fine here